### ISSUE TITLE
Auto-Leave Voice Channel Delay (Seconds) is not in settings.json by default

### ISSUE THREAD

**Issue by TheMonDon** [RobertBorghese/DBM#22]
As per the message I sent on discord, this setting has a default value of 10 but is not actually in settings.json by default. It does seem to save once you manually change it and save your project though.

**Comment by RobertBorghese** [comment]
Has there been any issues with this? I think the preferred fix would be to have a default value used in the bot.js if not available.

**Comment by RobertBorghese** [comment]
Yeah, right here: https://github.com/RobertBorghese/DBM/blob/a0c89a079fe40ff49d2137bc8d8f3b15f14814fd/bot.js#L3250

It should be fine... maybe??

**Comment by RobertBorghese** [comment]
Could prob fix in editor too, but how's this? [09accdc](https://github.com/RobertBorghese/DBM/commit/09accdc8f49e3266aaee3762d41a557833918645)

**Comment by TheMonDon** [comment]
> Could prob fix in editor too, but how's this? [09accdc](https://github.com/RobertBorghese/DBM/commit/09accdc8f49e3266aaee3762d41a557833918645)

I suppose that would work, as long as you also make it so it's 0 when a new project is created inside the editor

**Comment by TheMonDon** [comment]
> Has there been any issues with this? I think the preferred fix would be to have a default value used in the bot.js if not available.

Not really any issue, but I made a music mod that used it and it's just that it wasn't a property and it wasn't inside settings.json which felt like a bug
